projects/azure/acs-engine/azure-purge/is-artifact.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: azure-purge-artifact
```

Comment:
We don't need to separate image streams, elsewhere we use the same but push different tags (`binaries` and `latest`).